app/models/vm_or_template/operations/relocation.rb

```diff
@@ -2,8 +2,8 @@ module VmOrTemplate::Operations::Relocation
   extend ActiveSupport::Concern
 
   included do
-    supports_not :live_migrate, :reason => _("Operation not supported.")
-    supports_not :evacuate, :reason => _("Operation not supported.")
+    supports_not :live_migrate, :reason => "Operation not supported."
```

Comment:
this and the other `_()` thing is unrelated, but is need for this. 
Because `settings.rb` now triggers loading all subclasses and `_()` does not work without settings